Library/Formula/lnav.rb

```diff
@@ -21,6 +21,7 @@ class Lnav < Formula
 
   depends_on "readline"
   depends_on "pcre"
+  depends_on "curl"
```

Comment:
I don't think the system curl has sftp support, which is useful in this context.